lib/template/config/initializers/rollbar.rb

```diff
@@ -1,6 +1,7 @@
 Rollbar.configure do |config|
   config.enabled = ENV.has_key?('ROLLBAR_ACCESS_TOKEN')
   config.access_token = ENV["ROLLBAR_ACCESS_TOKEN"]
+  config.environment = ENV['ROLLBAR_ENVIRONMENT']
```

Comment:
Yeah, it makes sense. Thanks for explanation! :+1:

I wasn't sure how Rollbar gem acts when `environment` is `nil`, but seems like it [handles everything pretty well](https://github.com/rollbar/rollbar-gem/blob/d06b18d2a98d8effdde605355ebe14add7ae5d3d/lib/rollbar.rb#L368-L369) :smile: